apps/pipelines/views.py

```diff
@@ -12,6 +15,39 @@ def pipeline_builder(request, team_slug: str):
     return render(request, "pipelines/pipeline_builder.html")
 
 
+def pipeline_node_input_types(request, team_slug):
+    """Returns all the input types for the various nodes
+
+    Example:
+        {
+          "CreateReport": {
+            "prompt": "<class 'str'>"
```

Comment:
as mentioned above, I think this should also have the inputs of `LLMResponse`

Reply:
Yes! I've now removed this docstring since the way I'm passing this to the frontend is different.